source/common/config/grpc_mux_impl.cc

```diff
@@ -206,5 +220,45 @@ void GrpcMuxImpl::handleEstablishmentFailure() {
   }
 }
 
+void GrpcMuxImpl::populateControlPlaneInfo(const envoy::api::v2::DiscoveryResponse& message) {
```

Comment:
FWIW, there is a bunch of refactoring going on here in https://github.com/envoyproxy/envoy/pull/6430 and followup PRs. Might be some fun merge issues :)